pkg/model/job.go

```diff
@@ -44,10 +47,8 @@ type Job struct {
 
 	// All local events associated with the job
 	LocalEvents []JobLocalEvent `json:"LocalJobEvents,omitempty"`
-}
 
-func (job Job) String() string {
-	return job.ID
```

Comment:
Removed this method because it'd prevent me from printing the full Job object in logs. Plus, it had zero references.